MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -1431,9 +1431,12 @@ public static void processScenarioCompletion(ResolveScenarioTracker rst) {
                     StratconFacility facility = track.getFacility(scenario.getCoords());
 
                     boolean victory = rst.getScenario().getStatus().isOverallVictory();
+                    boolean draw = rst.getScenario().getStatus().isDraw();
 
                     if (scenario.isRequiredScenario()) {
```

Comment:
I would suggest just doing "isDraw" as part of this conditional rather than mucking about with adding/removing extra victory points.

Reply:
I'll have that done first thing, tomorrow

Reply:
Done